bundle/Resources/views/Content/form_fields.html.twig

```diff
@@ -0,0 +1,42 @@
+{% use 'form_div_layout.html.twig' %}
+
+{% block form_errors %}
+    {%- for error in errors if errors|length > 0 -%}
+        <em class="ez-field-edit-error">{{ error.message }}</em>
+    {%- endfor -%}
+{% endblock form_errors %}
+
+{% block form_label %}
+    {%- if form.value is defined -%}
+        {% set label_attr = label_attr|merge({'for': form.value.vars.id}) %}
+    {%- endif %}
+    {%- if compound is same as(false) -%}
+        {% set label_attr = label_attr|merge({'class': 'ez-sub-field-name'}) %}
+    {%- endif %}
+    {{- parent('form_label') -}}
+{% endblock form_label %}
+
+{%- block form_widget_compound -%}
+    <fieldset>
+        {%- if form.parent is empty -%}
+            {{ form_errors(form) }}
```

Comment:
validation errors are not supposed to appear here but in `div.ez-field-edit-text-zone` (as written in https://github.com/ezsystems/repository-forms/pull/134/files#diff-6972ff935106f8fc8461d54811ad45caR21). That said, if we have errors dedicated to *sub fields*, it should appear in the corresponding `div.ez-sub-field-text-zone`